index.html

```diff
@@ -33,52 +33,219 @@
+                </div>
+                <div class="des_text">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.
+                </div>
+                <div class="tag_text">
+                    <ul>
+                        <li class="tags">html</li>
+                        <li class="tags">css</li>
+                        <li class="tags">javascript</li>
+                    </ul>
+                </div>
+                <button type="button" class="button">See Project</button>
+            </div>
+            <div class="project1">
+                <img class="img" src="assets/images/SnapshootPortfolio(1).png" alt="Project image">
+                <div class="pri_text">
+                    <h3>Tonic</h3>
+                </div>
+                <div class="det_text">
+                    <h3 class="canopy">CANOPY</h3>
+                    <img src="assets/images/Counter.png" alt="Counter">
+                    <h3 class="detail">BACKEND DEV</h3>
+                    <img src="assets/images/Counter.png" alt="Counter">
+                    <h3 class="detail">2015</h3>
+                </div>
+                <div class="des_text">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.
+                </div>
+                <div class="tag_text">
+                    <ul>
+                        <li class="tags">html</li>
+                        <li class="tags">css</li>
+                        <li class="tags">javascript</li>
+                    </ul>
+                </div>
+                <button type="button" class="button">See Project</button>
+            </div>
+            <div class="project1">
+                <img class="img" src="assets/images/SnapshootPortfolio(2).png" alt="Project image">
+                <div class="pri_text">
+                    <h3>Multi-Post Stories</h3>
+                </div>
+                <div class="det_text">
+                    <h3 class="canopy">CANOPY</h3>
+                    <img src="assets/images/Counter.png" alt="Counter">
+                    <h3 class="detail">BACKEND DEV</h3>
+                    <img src="assets/images/Counter.png" alt="Counter">
+                    <h3 class="detail">2015</h3>
+                </div>
+                <div class="des_text">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.
+                </div>
+                <div class="tag_text">
+                    <ul>
+                        <li class="tags">html</li>
+                        <li class="tags">css</li>
+                        <li class="tags">javascript</li>
+                    </ul>
+                </div>
+                <button type="button" class="button">See Project</button>
+            </div>
+            <!-- work section end -->
+
+
+        </section>
+        <div class="about">
```

Comment:
- [ ] You did a great job enclosing the works in a `<section>` tag, how about you use it for the about section as well? This will improve your website accessibility and search engine optimization!